### ISSUE TITLE
Editable Reblogs extension not available in gallery

### ISSUE THREAD

**Issue by bvtsang** [hobinjk/XKit#83]
According to [chooope](https://github.com/hobinjk/XKit/issues/64#issuecomment-126854114) in #64:

> Not sure if this is the right place to say this or not, but uh-- the editable reblogs extension just isn't showing up for me? At all? I've got new xkit, everything seems to be up to date with it, but when I search for this extension in the gallery it just isn't there. Am I doing something wrong or...?

When asked if the old XKit was uninstalled, disabled, or non-existent:

> No, old xkit is there! I did see the post on the blog about old xkit needing to be uninstalled but when I did that xkit just disappeared altogether, and the about:config solution didn't help, so I reinstalled it since everything else works fine, aside from my aforementioned issue...

**Comment by nightpool** [comment]
I'm guessing that the situation is that @chooope is *actually* interacting
with the old xkit. (because when they uninstalled old xkit, their settings
button disappeared)

**Comment by bvtsang** [comment]
@chooope, just to confirm, did you install XKit from https://addons.mozilla.org/en-US/firefox/addon/new-xkit?

**Comment by KaeganDeoir** [comment]
I'm using Chrome and I'm having the same issue. This is the first time I've ever downloaded XKit, and I got it from the google app store following the link that was on the XKit website.

**Comment by tobytalker** [comment]
I'm having this issue too. This is my first time installing XKit, and I have the newst update, but the 'Editable Reblogs' extension is just not showing up in the gallery.

**Comment by nightpool** [comment]
can you please post your xkit version?
![](https://cloud.githubusercontent.com/assets/233815/9022978/2935a1b4-384f-11e5-9521-2c7db5884309.png)

**Comment by KaeganDeoir** [comment]
I fixed my problem this morning. I had to update the Chrome browser, then updated the XKit patches. My version was 7.6.5

**Comment by chooope** [comment]
Hi, sorry for the delay! @bvtsang I did download from that link, yes, and I am on Firefox. This is my version of xkit. http://i.gyazo.com/d4e3b71d2b39680d253aec0c9aa8d7ec.png

**Comment by nightpool** [comment]
@chooope that version is an out-of-date old xkit version. Uninstall all xkit extensions in about:addons (ctrl-shift-a), and reinstall new xkit from https://addons.mozilla.org/en-US/firefox/addon/new-xkit/

**Comment by bvtsang** [comment]
Closing this issue because it seems that updating XKit fixes it.

Though now that I think about it, it seems easy for people to get confused by the old and new XKits. We could fix this with better education on the new XKit blog or changing URLs (like in c0b093c93f6aea546eabd213c1e5ac2fc6758c73).

**Comment by chooope** [comment]
Hey, uh, @nightpool and @bvtsang , when I do that I don't see xkit at all anymore? Like the xkit button is gone from my dash/activity page and none of the settings are there anymore.

**Comment by bvtsang** [comment]
I just tried the following steps:

0. Uninstall all XKit extensions (I normally don't use Firefox, so I have no installed extensions)
  - You can follow @nightpool's steps in his comment above to do this
1. Log into Tumblr on Firefox (I'm on Firefox 39)
2. Visit https://addons.mozilla.org/en-US/firefox/addon/new-xkit/ in a new tab
3. Click yellow striped Add to Firefox button (version listed on that page is currently 7.6.7)
4. In the caution popup, click blue Add to Firefox button
5. In the Software Installation popup, click Install Now
6. Back on the tab with Tumblr, click refresh
7. Click Refresh page and start using XKit
8. Click XKit button
9. Click Cancel Tour
10. Click XKit button again
11. Click Get Extensions tab
12. Expect to see Editable Reblogs in gallery

And I see the Editable Reblogs extension. Can you try these steps? Because I can't reproduce this issue.

I'm reopening this issue and will close it when @chooope confirms that it's fixed.

**Comment by chooope** [comment]
SO I FEEL LIKE AN IDIOT BUT IT TURNS OUT the problem was that I was on an older version (like 36) of Firefox lmao. Which I did intentionally because old xkit had broken on newer Firefox updates. I just, totally forgot about it until you said which version you were using and when I updated Firefox it worked just fine (whereas previously, when I got to step 7, xkit just didn't show up for me on tumblr. like it wasn't there)

SO YEAH we're all good now. if anyone else ever has this problem, tell them to check what version of firefox they're using omg. apologies for all of this!
